components/sections/earn-section/index.tsx

```diff
@@ -0,0 +1,225 @@
+import { cn } from "@/lib/utils";
+import Image from "next/image";
+import Link from "next/link";
+import { ReactElement } from "react";
+
+interface Card {
```

Comment:
Why do you call it "Card" is it a generic type? But I think it only refers to earn cards, maybe we should use an appropriate name